casper/src/main/scala/coop/rchain/casper/engine/CasperLaunch.scala

```diff
@@ -20,121 +19,114 @@ import coop.rchain.comm.transport.TransportLayer
 import coop.rchain.metrics.{Metrics, Span}
 import coop.rchain.shared._
 
-object CasperLaunch {
+trait CasperLaunch[F[_]] {
```

Comment:
This new `trait` isn't used anywhere. It's not really a TF algebra, just some launcher. I don't understand the context of this change.

Reply:
Well, for one thing, it eliminated a few sets of long context bounds. Please take a look at my comments - I think we share the same opinion

Reply:
Good point on the trait not being used.

It is a TF algebra, no matter how trivial and unused :)

The context is: I wanted TF further down in casper, and figured it's easiest to introduce top-down. CasperLaunch was closest to `main` that wasn't TF, so I started from there.

Ignoring the above, the change makes a couple methods private, removes the unneeded CasperInit wrapper, and deduplicates the deps lists as pointed out by @marcin-rzeznicki.